### PR TITLE
[docs] chore: update Sui CLI documentation

### PR DESCRIPTION
**Pull request by stillmatic**
mainly updating outdated names that I ran across

### CODE REVIEW

File: doc/src/build/cli-client.md [hunk 1]
```diff
@@ -245,9 +248,13 @@ override this setting by providing a path to the directory where this
 file is stored:
 
 ```shell
-$ sui console --config /path/to/client/config/file
+$ sui console --client.config /path/to/client/config/file
 ```
 
+Within the console, running a command such as `addresses` returns the output of
+`sui client addresses`, for the currently defined client configuration. This is
```
Comment (randall-Mysten) [inline doc/src/build/cli-client.md:255]:
```suggestion
`sui client addresses` for the currently defined client configuration. This is
```

File: doc/src/build/cli-client.md [hunk 2]
```diff
@@ -209,17 +210,19 @@ The following commands are supported by the Sui client:
...
-    new-address           Generate new address and keypair
-    object                Get obj info
+    new-address           Generate new address and keypair with keypair scheme flag {ed25519
+                              | secp256k1}
+    object                Get object info
     objects               Obtain all objects owned by the address
     publish               Publish Move modules
     split-coin            Split a coin object into multiple coins
-    switch                Switch active address and network (e.g., Devnet, local RPC server)
+    switch                Switch active address and network(e.g., devnet, local rpc server)
```
Comment (randall-Mysten) [inline doc/src/build/cli-client.md:219]: this is Devnet and RPC Server in the UI, should it match?

**Review by randall-Mysten — APPROVED** [review]
Thanks for making the doc update. Added one suggested change.

File: doc/src/build/cli-client.md (hunk 2 above)
Comment (stillmatic) [reply doc/src/build/cli-client.md:219]: on `--- Sui Console 0.9.0-df05544f ---`

image

looks like this is from https://github.com/MystenLabs/sui/blob/2bf1304757e212f337de076a5cfea29dd0f61aba/crates/sui/src/client_commands.rs#L52. happy to make the change there, a cursory glance through `git blame` suggests that the readme's phrasing was never committed (publicly, at least)

**Comment by randall-Mysten** [comment]
Please reopen if this is still a relevant issue. Closing stale PRs

**Pull request closed without merging**